src/cephadm/cephadm

```diff
@@ -1510,6 +1514,7 @@ def command_bootstrap():
         os.fchmod(f.fileno(), 0o600)
         f.write(config)
 
+    make_var_run(fsid, uid, gid)
```

Comment:
Maybe I missed something, but outside `command_adopt`, why not create this dir along with the others in `create_daemon_dirs`?

https://github.com/ceph/ceph/blob/e4c1c0e610209fe6423a063bed73a9e48fcb266b/src/cephadm/cephadm#L717

Reply:
Perhaps `command_adopt` should be using `create_daemon_dirs` too?

Reply:
yeah maybe/probably. 

we can't rely on create-daemon_dirs in the general case though because it's called by deploy_daemon which is passed in a container object, which is built by get_container and get_container_mounts, which needs the /var/run dir to already exist.

i looked at instead making the get_container_mounts create it if it is missing but it doesn't have the uid/gid handy.  all of the get_container callers do have it except run (don't care) and enter (sort of care... it doesn't make sense here i think to start a container to extract the right uid/gid to create the run dir when it should already exist at this point)... the only passing it through if the dir already exists is i think the right behavior.

anyway, i think we leave adopt for a future possible cleanup